src/encoded/static/components/filegallery.js

```diff
@@ -1879,6 +1879,9 @@ function filterItems(array, key, keyValue) {
             if (keyValue[0] === 'All assemblies') {
                 return true;
             }
+            if (el.genome_annotation) {
+                return (`${el[key]} ${el.genome_annotation}` === keyValue[0]);
+            }
```

Comment:
I’m not proposing changing this at this stage, but I do get kind of nervous about generating a string to check a property of an object against a displayed value. If this goes through a refactor in the future it might be nice to just be able to compare annotation and assembly against both values to see if it passes the filter. But again, I wouldn’t recommend changing this at this stage — I can see how it works, and I’m not even sure my discomfort is appropriate here.

I can see similar things happening below, using a generated string to get an element of assembly.